pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md

```diff
@@ -111,7 +109,134 @@ Before you begin, you must:
       - kind: ConfigMap
          name: node-feature-discovery
       strategy: ApplyOnce
+    ```
+
+    Konvoy defines the selectors and sets the correct labels on the Cluster objects. For a more detailed explanation of how ClusterResourceSets work, see the [Extension Proposal][clusterresourceset_caep].
+
+[install_docker]: https://docs.docker.com/get-docker/
+[install_clusterawsadm]: https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases
+[install_kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+[aws_credentials]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+[capa]: https://github.com/kubernetes-sigs/cluster-api-provider-aws
+[kind]: https://github.com/kubernetes-sigs/kind
+[capi_book]: https://cluster-api.sigs.k8s.io/
+[calico]: https://docs.projectcalico.org/
+[capi]: https://github.com/kubernetes-sigs/cluster-api/tree/v0.3.20/
+[kcp]: https://github.com/kubernetes-sigs/cluster-api/tree/v0.3.20/controlplane/kubeadm
+[cabpk]: https://github.com/kubernetes-sigs/cluster-api/tree/v0.3.20/bootstrap/kubeadm
+[clusterresourceset_caep]: https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20200220-cluster-resource-set.md
+[prerequisites]: ../prerequisites
+=======
+---
+layout: layout.pug
+navigationTitle: Bootstrap
+title: Bootstrap
+menuWeight: 15
+excerpt: Prepare to deploy Kubernetes clusters
+enterprise: false
+---
+
+To create Kubernetes clusters, Konvoy uses [Cluster API][capi_book] (CAPI) controllers. These controllers run on a Kubernetes cluster. To get started, you need a _bootstrap_ cluster. By default, Konvoy creates a bootstrap cluster for you in a Docker container using the Kubernetes-in-Docker ([KIND][kind]) tool.
```

Comment:
Where did this information come from? I'm not sure what we're doing here.

Reply:
This info isn't in the published docs, so it slipped into your PR somehow. Can we fix it up?

https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/

Reply:
this is ok in the raw file.